Source/FakeItEasy.Tests/Core/DynamicContainerTests.cs

```diff
@@ -106,22 +111,52 @@ public void Should_not_fail_when_more_than_one_configurator_exists_for_a_given_t
 
         private DynamicContainer CreateContainer()
         {
-            return new DynamicContainer(this.availableDummyFactories, this.availableConfigurators);
+            return new DynamicContainer(this.availableDummyFactories, this.availableOptionsBuilders);
         }
 
-        public class ConfigurationForTypeWithDummyFactory : FakeConfigurator<TypeWithDummyFactory>
+        public class OptionsBuilderForTypeWithDummyFactory : IFakeOptionsBuilder

```

Comment:
Why implement the interface directly instead of inheriting `FakeOptionsBuilder<TypeWithDummyFactory>`?